consensus/checks.go

```diff
@@ -11,10 +11,6 @@ import (
 const MaxBlockNumDiff = 100
 
 func (consensus *Consensus) validatorSanityChecks(msg *msg_pb.Message) bool {
-	if msg.GetConsensus() == nil {
```

Comment:
Why remove this check ? The consensus msg can be nil. We hit that nilpointer before, that’s why we added the check.

Reply:
Because it’s handled at message validation earlier

Reply:
And because there is way more compute available to handle this rather than force the consensus thread to waste compute